CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## main
 
+### Features ✨ and improvements 🏁
+* Added the new `moveStyleLayer(layedId: String, layerPosition: LayerPosition?)` API. ([#588](https://github.com/mapbox/mapbox-maps-ios/pull/588))
```

Comment:
Can we hold off on adding this entry, until we've migrated our existing `_moveLayer` function?